tests/lib/rules/no-await-sync-events.test.ts

```diff
@@ -168,15 +175,17 @@ ruleTester.run(RULE_NAME, rule, {
     },
 
     // valid tests for fire-event when only user-event set in eventModules
-    ...FIRE_EVENT_FUNCTIONS.map((func) => ({
-      code: `
-        import { fireEvent } from '@testing-library/framework';
-        test('should not report fireEvent.${func} sync event awaited', async() => {
-          await fireEvent.${func}('foo');
-        });
-      `,
-      options: [{ eventModules: ['user-event'] }],
-    })),
+    ...SUPPORTED_TESTING_FRAMEWORKS.flatMap((testingFramework) =>
+      FIRE_EVENT_FUNCTIONS.map((func) => ({
+        code: `
+          import { fireEvent } from '@testing-library/${testingFramework}';
```

Comment:
Shouldn't this be the following?
Otherwise, we end up with `@testing-library/@testing-library/dom`

```suggestion
          import { fireEvent } from '${testingFramework}';
```